packages/typespec-ts/test/modularUnit/apiOperations.spec.ts

```diff
@@ -628,11 +709,13 @@ describe("api operations in Modular", () => {
       `;
       const operationFiles = await emitModularOperationsFromTypeSpec(
         tspContent,
-        false,
-        true,
-        false,
-        false,
-        true
+        {
```

Comment:
Really appreciate this refactoring, this will reduce efforts if any changes happen in options.